src/Domain/EmbeddingMaps/AffineMap.hpp

```diff
@@ -14,19 +14,27 @@
 
 namespace EmbeddingMaps {
```

Comment:
Yes, that sounds right.  I just wanted to make sure we weren't making the choice based on an argument that doesn't apply here.